include/script_api/macros.h

```diff
@@ -679,19 +680,6 @@
         End \
     }
 
-// alternate version of EVT_EXIT_WALK which includes a call to DisablePlayerInput
-#define EVT_EXIT_WALK_FIXED(walkDistance, exitIdx, map, entryIdx) \
-    { \
-        SetGroup(EVT_GROUP_1B) \
-        Call(DisablePlayerInput, TRUE) \
-        Call(UseExitHeading, walkDistance, exitIdx) \
-        Exec(ExitWalk) \
-        Call(GotoMap, Ref(map), entryIdx) \
-        Wait(100) \
-        Return \
-        End \
-    }
-
```

Comment:
IDK about this. LZS is cool!

Reply:
Either way I'd rather not have an extra definition for this used on only two maps.